src/components/form-fields/combobox/combobox-form-field.tsx

```diff
@@ -29,18 +29,21 @@ export function ComboboxFormField<TFormModel extends FieldValues, TFieldModel>({
     <FormField
       control={control}
       name={name}
-      render={({ field }) => (
-        <ComboboxFormItem
-          label={label}
-          description={description}
-          required={required}
-          buttonClassName={buttonClassName}
-          onSelect={field.onChange}
-          fieldValue={field.value}
-          inForm={true}
-          {...props}
-        />
-      )}
+      render={({ field }) => {
+        const _fieldValue = props.initialValues ?? field.value;
```

Comment:
In ComboboxFormField initial values need to come from form one way or another, not explicit initialValues. If something needs to supply values directly to the Combobox, they should use ComboboxFormItem or just Combobox.